### PR TITLE
Feature: adding a translation for a js/ru page

### LINKED ISSUE #219: Translation js/fe course page
- Change title. (e.g. Javascript / Frontend course RU)

- Translate communication block

- Check all texts

### PR DESCRIPTION
**Pull request by NikitaStarmoussov**
## What type of PR is this? (select all that apply)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Description

Corrected the translation for the js/ru page

## Related Tickets & Documents

- Related Issue #219
- Closes #219

## Screenshots, Recordings

_Please replace this line with any relevant images for UI changes._

## Added/updated tests?

- [x] 👌 Yes
- [ ] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

### CODE REVIEW

**Review by ansivgit — APPROVED** [review]

File: src/features/communication/communication.tsx [hunk 1]
```diff
@@ -7,29 +7,61 @@ import './communication.scss';
...
+    secondParagraphSecondHalf:
+      'Here you could find rules, links to other courses and Telegram channels related to your country or even city. Feel free to use it :)',
+  },
+  ru: {
+    title: 'Коммуникации',
+    subTitle: 'Дискорд — основной способ коммуникации в RS School',
+    firstParagraphFirstHalf: 'Здесь есть ссылка на',
+    discordLink: 'дискорд сервер курса',
+    firstParagraphSecondHalf:
+      'где вы можете посмотреть последние новости и общаться со студентами.',
```
Comment (natanchik) [inline src/features/communication/communication.tsx:31]: Maybe we should add `, ` (comma and space) before text

File: src/features/communication/communication.tsx [hunk 2]
```diff
@@ -7,29 +7,61 @@ import './communication.scss';
...
+    secondParagraphFirstHalf: 'Более подробная информация об общении в RS School в',
+    rsDocsLink: 'RS Docs',
+    secondParagraphSecondHalf:
+      'где вы можете найти правила, ссылки на другие курсы и каналы Telegram, связанные с вашей страной или даже городом. Не стесняйтесь использовать их :)',
```
Comment (natanchik) [inline src/features/communication/communication.tsx:35]: And here `, `

**Review by natanchik — APPROVED** [review]
A few suggestions for minor edits

File: src/features/communication/communication.tsx [hunk 3]
```diff
@@ -7,29 +7,61 @@ import './communication.scss';
...
+    firstParagraphSecondHalf: 'where you can see latest news and chat with students.',
+    secondParagraphFirstHalf: 'More info about communication at RS School in the',
+    rsDocsLink: 'RS Docs',
+    secondParagraphSecondHalf:
+      'Here you could find rules, links to other courses and Telegram channels related to your country or even city. Feel free to use it :)',
+  },
+  ru: {
+    title: 'Коммуникации',
+    subTitle: 'Дискорд — основной способ коммуникации в RS School',
+    firstParagraphFirstHalf: 'Здесь есть ссылка на',
```
Comment (Tedzury) [inline src/features/communication/communication.tsx:28]:
```suggestion
    firstParagraphFirstHalf: 'Вот ссылка на',
```

File: src/features/communication/communication.tsx [hunk 4]
```diff
@@ -7,29 +7,61 @@ import './communication.scss';
...
+    discordLink: 'course discord server',
+    firstParagraphSecondHalf: 'where you can see latest news and chat with students.',
+    secondParagraphFirstHalf: 'More info about communication at RS School in the',
+    rsDocsLink: 'RS Docs',
+    secondParagraphSecondHalf:
+      'Here you could find rules, links to other courses and Telegram channels related to your country or even city. Feel free to use it :)',
+  },
+  ru: {
+    title: 'Коммуникации',
+    subTitle: 'Дискорд — основной способ коммуникации в RS School',
```
Comment (Tedzury) [inline src/features/communication/communication.tsx:27]:
```suggestion
    subTitle: 'Дискорд — основной способ общения в RS School',
```

File: src/features/communication/communication.tsx (hunk 2 above)
Comment (Tedzury) [inline src/features/communication/communication.tsx:35]:
```suggestion
      'где вы можете найти правила, ссылки на другие курсы и каналы Telegram, связанные с вашей страной или даже городом. Не стесняйтесь пользоваться ими :)',
```

File: src/features/communication/communication.tsx [hunk 5]
```diff
@@ -7,29 +7,61 @@ import './communication.scss';
...
             <Paragraph>
-              Here is link for the{' '}
-              <ExternalLink href={DISCORD_LINKS[courseName]} text="course discord server" /> where
-              you can see latest news and chat with students.
+              {localizedContent[lang].firstParagraphFirstHalf}{' '}
+              <ExternalLink
+                href={DISCORD_LINKS[courseName]}
+                text={localizedContent[lang].discordLink}
+              />
+              {localizedContent[lang].firstParagraphSecondHalf}
```
Comment (Tedzury) [inline src/features/communication/communication.tsx:56]:
```suggestion
              {' '}{localizedContent[lang].firstParagraphSecondHalf}
```

File: src/features/communication/communication.tsx [hunk 6]
```diff
@@ -7,29 +7,61 @@ import './communication.scss';
...
+    firstParagraphFirstHalf: 'Here is link for the',
+    discordLink: 'course discord server',
+    firstParagraphSecondHalf: 'where you can see latest news and chat with students.',
+    secondParagraphFirstHalf: 'More info about communication at RS School in the',
+    rsDocsLink: 'RS Docs',
+    secondParagraphSecondHalf:
+      'Here you could find rules, links to other courses and Telegram channels related to your country or even city. Feel free to use it :)',
+  },
+  ru: {
+    title: 'Коммуникации',
```
Comment (Tedzury) [inline src/features/communication/communication.tsx:26]:
```suggestion
    title: 'Общение',
```

**Review by andron13 — APPROVED** [review]

**Review by dzmitry-varabei — APPROVED** [review]

File: src/app/services/data/js-ru-path.data.ts [hunk 7]
```diff
@@ -16,7 +16,7 @@ export const jsPathRu: JSPath[] = [
     id: 2,
     title: 'Этап 2',
     description:
-      'Чтобы пройти во второй этап, вам необходимо успешно пройти задачи и тесты первого этапа без пропусков сроков и пройти пробный технический интервью с одним из наших менторов. Второй этап длится 20 недель. Вам будет назначен личный ментор, который будет отвечать на ваши вопросы. На этом этапе включаются практические задания и тесты, которые будут проверяться и оцениваться вашим ментором, а также через методы перекрестной проверки. Помимо этого, проходят проверку и оценка мок-интервью разных менторов.',
+      'Чтобы перейти на второй этап, вам необходимо успешно выполнить задания и тесты первого этапа без пропуска сроков и пройти пробное техническое интервью с одним из наших менторов. Второй этап длится 20 недель. Вам будет назначен личный ментор, который будет отвечать на ваши вопросы. Этот этап включает в себя практические задания и тесты, которые будут проверяться и оцениваться вашим наставником, а также методом перекрестной проверки. Помимо этого, проводятся мок-интервью у разных менторов.',
```
Comment (ansivgit) [inline src/app/services/data/js-ru-path.data.ts:19]:
```suggestion
      'Чтобы перейти на второй этап, вам необходимо успешно выполнить задания и тесты первого этапа без пропуска сроков и пройти пробное техническое интервью с одним из наших менторов. Второй этап длится 20 недель. Вам будет назначен личный ментор, который будет отвечать на ваши вопросы. Этот этап включает в себя практические задания и тесты, которые будут проверяться и оцениваться вашим ментором, а также перекрестной проверкой других студентов. Помимо этого, проводятся пробные интервью с другими менторами.',
```

File: src/app/services/data/js-ru-path.data.ts [hunk 8]
```diff
@@ -31,7 +31,7 @@ export const jsPathRu: JSPath[] = [
     id: 3,
     title: 'Этап 3',
     description:
-      'Обучение использованию React или Angular (выбор принадлежит ученику). Для вступления в обучение необходимо успешно пройти два этапа обучения. Формат: менторство, самостоятельное обучение, вебинары, общение на Discord. Практические занятия ревьюют и оценивают менторы, так же как и через методы перекрестной проверки. Протяжении обучения проводятся мок-интервью с разными менторами.',
+      'Обучение применению React или Angular (выбор за студентом). Чтобы записаться на курс, необходимо успешно пройти два этапа обучения. Формат: менторство, самостоятельные занятия, вебинары и общение в Discord. Практические занятия разбираются и оцениваются наставниками, а также методом перекрестной проверки. На протяжении всего обучения проводятся мок-интервью у разных менторов.',
```
Comment (ansivgit) [inline src/app/services/data/js-ru-path.data.ts:34]:
```suggestion
      'Обучение применению React или Angular (выбор за студентом). Чтобы записаться на курс, необходимо успешно пройти два этапа обучения. Формат: менторство, самостоятельные занятия, вебинары и общение в Discord. Практические занятия разбираются и оцениваются наставниками, а также перекрестной проверкой другими студентами. На протяжении всего обучения проводятся пробные интервью с другими менторами.',
```

**Review by ansivgit — APPROVED** [review]

**Pull request merged**